docs/en/defined-terms.rst

```diff
@@ -51,6 +51,8 @@ Below are the description of acronyms and definitions which are useful for furth
      - An architectural component that enables IT Wallet system participants to establish trust, in terms of reliability and compliance of all participants with the regulatory framework governing the digital identity system.
    * - Level of Assurance
      - The degree of confidence in the vetting process used to establish the identity of the User and the degree of confidence that the User who presents the credential is the same User to whom the credential was issued.
+   * - Holder Key Binding
+     - Ability of the Holder to prove legitimate possession of the private part, related to the public part attested by a Trusted Third Party.
```

Comment:
It is a very minor comment. 
However, I recommend for clarity substituting the ``private part`` with ``private key`` and ``public part`` with ``public key``.

Reply:
Would it be valuable to add the definition of the `Trusted Third Party` as the following deriving from [FIPS 186-5](https://doi.org/10.6028/NIST.FIPS.186-5), or [NIST SP 800-89](https://doi.org/10.6028/NIST.SP.800-89):

- `An entity other than the key pair owner (Holder) and Verifier that is trusted by the Holder or the Verifier or both. Sometimes shortened to “Trusted Party.`
- ` An entity other than the Holder and Verifier that is trusted by the Holder, the Verifier or both to provide certain services.`

Reply:
Ok, see my suggestion